### PR TITLE
Add El greeting

### PR DESCRIPTION
**Pull request by Bonesbhbhbh**
Add greeting for El

### CODE REVIEW

File: src/main/java/hellos/Hellos.java [hunk 1]
```diff
@@ -9,6 +9,7 @@ public class Hellos {
   public String generateOutput() {
     StringBuilder builder = new StringBuilder();
 
+    builder.append(elSaysHello());
```
Comment (JosieBarber) [inline src/main/java/hellos/Hellos.java:12]: Why would you do that :(

**Review by JosieBarber — DISMISSED** [review]

**Review by amoak017 — DISMISSED** [review]

File: src/main/java/hellos/Hellos.java (hunk 1 above)
Comment (Bonesbhbhbh) [reply src/main/java/hellos/Hellos.java:12]: because i wanted to say hi :,(

Comment (JosieBarber) [inline src/main/java/hellos/Hellos.java:12]: Why'd you do this again D:

**Review by JosieBarber — APPROVED** [review]

**Review by amoak017 — APPROVED** [review]
very good change here, I apporve

**Pull request merged**